### ISSUE TITLE
makedvd command doesn't accept path names w/ spaces

### ISSUE THREAD

**Issue by grepper** [grepper/tovid#48]
_From [mtromp...@gmail.com](https://code.google.comNone) on March 29, 2008 02:42:46_

tovid successfully encoded.  But when trying to burn with makedvd, path
names w/ spaces doesn't work.

e.g. makedvd -quiet -author -burn -device /dev/dvdrw /home/mike/12 Hours Of
Night/12_Hours_of_Night.xml

gives the error: 

Error: Unrecognized command-line option '/home/mikee/12'

_Original issue: http://code.google.com/p/tovid/issues/detail?id=46_

**Comment by grepper** [comment]
_From [pengi.fi...@gmail.com](https://code.google.com/u/110441193590422221193/) on March 29, 2008 00:54:55_

heh, this isn't a tovid bug. Use quotes or a backslash to use spaces in path/file
names. See the 'Quoting' section of the bash man page.

**Status:** Invalid

**Comment by grepper** [comment]
_From [mtromp...@gmail.com](https://code.google.comNone) on March 29, 2008 12:03:40_

Thanks for the tip.  Just started using linux and was using cli for tovid.  I'm able
to cd into /path with spaces in /path, but didn't know makedvd wouldn't pick it up. 
Thanks.

I did a workaround for this.  I copied files from the original folder into another
folder w/ no spaces, edited the generated xml to point to the new folder, and ran
makedvd.  It worked fine.
